xml/System.IO/BinaryReader.xml

```diff
@@ -744,10 +746,12 @@
         <Parameter Name="buffer" Type="System.Span&lt;System.Char&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="buffer">A span of characters. When this method returns, the contents of this region are replaced by the characters read from the current source.</param>
+        <summary>Reads, from the current stream, the same number of characters as the length of the provided buffer, writes them in the provided buffer, and advances the current position in accordance with the <see langword="Encoding" /> used and the specific character being read from the stream.</summary>
```

Comment:
Very similar to calling [BinaryReader.ReadChars(int)](https://github.com/dotnet/corefx/blob/65edbb39dd737e8529a343f385d2600e482b7ff3/src/Common/src/CoreLib/System/IO/BinaryReader.cs#L429) or [BinaryReader.Read(char[], int, int)](https://github.com/dotnet/corefx/blob/65edbb39dd737e8529a343f385d2600e482b7ff3/src/Common/src/CoreLib/System/IO/BinaryReader.cs#L322), because they call [InternalReadChars](https://github.com/dotnet/corefx/blob/65edbb39dd737e8529a343f385d2600e482b7ff3/src/Common/src/CoreLib/System/IO/BinaryReader.cs#L351):
https://github.com/dotnet/corefx/blob/65edbb39dd737e8529a343f385d2600e482b7ff3/src/Common/src/CoreLib/System/IO/BinaryReader.cs#L345